change/@microsoft-teams-js-795fcb3d-57f7-436b-b7e2-b3d3c2df6bad.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
```

Comment:
This should be type "none" since the docs only changes won't affect the behavior of the library at all.